geocode_lat_long/geocode_lat_long.md

```diff
@@ -0,0 +1,263 @@
+
+## Pinpointing Locations on Earth
+
+Before we can talk about locations all over the Earth, we have to establish the locations of a few very special points. The first two are the **North Pole** and the **South Pole**. If you imagine a line going straight through the planet connecting the two poles, this is the axis around which the Earth rotates creating night and day.
+
+The **equator** is the circle around the surface of the earth that is exactly halfway between the north and south poles. Any celestial body, like a planet or a star, that spins on an axis has two poles and an equator. Imagine a flat surface containing the equator and dividing the planet into a Northern Hemisphere and Southern Hemisphere. That flat surface is called the **equatorial plane**.
+
+A **meridian** is a straight line on the surface of the earth connecting the poles. The third point we need is the location of the [Greenwich Observatory](https://www.rmg.co.uk/royal-observatory) in London, England. The meridian running through the observatory is called the **Prime Meridian**. Unlike the poles, which are fixed points determined by the astronomical fact that the earth is spinning on an axis, the location of the Prime Meridian was a human decision made for [historical](https://greenwichmeantime.com/articles/history/navy/) rather than geographical reasons.
+
+Now that we have established the poles, the equator, and the prime meridian, we can use those to to create a coordinate system that will specify and point on earth!
+
+![On the left an image of Earth is shown from the side with the North Pole labeled at the top, the South Pole labeled at the bottom, and the equator represented as a dashed horizontal line around the middle. On the right Earth is shown from above so that the North Pole is visible. The prime meridian is represented by a dashed vertical line starting at the North Pole and continuing down through the United Kingdom, the western Mediterranean Sea, and West Africa.](media/poles_meridian_equator.svg)<!-- style = "max-width:800px;" border = 5px solid -->
+
+### Latitude and Longitude
+
+
+**Latitude** is a measure of how far north or south a location is from the equator. Given a point on the surface of the earth, imagine a straight line to the center of the earth. The angle that line makes with the equatorial plane, measured in degrees (denoted $ {}^\circ $), is the point's latitude. Since that angle could be either above or below the equatorial plane, we have to specify which angle we are talking about. The horizontal line on the picture below corresponding to 30 degrees north, or +30$ {}^\circ $, goes through the United States, while the line representing 30 degrees south, or -30$ {}^\circ $, runs through Chile and Argentina.
```

Comment:
Good call! What do you think about giving the degree symbol its own sentence?
```suggestion
**Latitude** is a measure of how far north or south a location is from the equator. Given a point on the surface of the earth, imagine a straight line to the center of the earth. The angle that line makes with the equatorial plane, measured in degrees, is the point's latitude. The symbol for degrees is the superscript circle $ {}^\circ $. 

Since the angle could be either above or below the equatorial plane, we have to specify which angle we are talking about. The horizontal line on the picture below corresponding to 30 degrees north, or +30$ {}^\circ $, goes through the United States, while the line representing 30 degrees south, or -30$ {}^\circ $, runs through Chile and Argentina.
```

Reply:
I think giving it its own sentence helps a lot